tests/test_main.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2018 Shehriyar Qureshi <SShehriyar266@gmail.com>
+import os
+import sys
+
+# add renpy directory to path else main module isn't found
+current_directory = os.getcwd()
+sys.path.insert(0, current_directory)
+
+# thou shalt not move this artifact unless thou can without errors
+import main
+
+
+class TestRenpyPath:
+    def test_renpy_path_to_renpy_base(self):
+
+        # We're just doing what the function does and comparing the values.
+        # This test should fail if the function is changed and returns
+        # something else.
+
+        # we change sys.argv to mimimc as if renpy is being run
+        sys.argv.pop(1)
+        sys.argv[0] = 'renpy.py'
+
+        renpy_base = main.path_to_renpy_base()
+
+        # we use sys.path[0] as it is the expected result of function
+        assert renpy_base == sys.path[0]
+
+    def test_path_to_common(self):
+        assert main.path_to_common(sys.path[0]) == "{}/renpy/common".format(
```

Comment:
Is this test relying on the previous testing having changed sys.path[0]? If so:
- Tests must be able to run in isolation.
- Like the previous test, it's using a modified sys.argv for no benefit.

Reply:
I don't think I get you. The second function is using `sys.path[0]`, not `sys.argv`. Is there a relation b/w `sys.argv` and `sys.path[0]` because `sys.path[0]` returns path to directory where the interpreter was called, while `sys.argv` is a list of command line arguments.

Upon testing `sys.path[0]` I realized the pytest command needs to be run in the renpy folder and not anywhere else because then `import main` throws `ImportError: no module named main` exception.

Reply:
I might have blinked and seen argv instead of path by mistake. I'll give this another look in a bit.

Reply:
Removed the use of sys.path[0] as it is not constant.